backend/de.metas.document.archive/de.metas.document.archive.base/src/main/java/de/metas/document/archive/DocOutboundUtils.java

```diff
@@ -65,8 +71,9 @@ public I_C_Doc_Outbound_Log_Line createOutboundLogLineRecord(@NonNull final I_C_
 		final DocStatus docStatus = documentBL.getDocStatusOrNull(reference);
 		docOutboundLogLineRecord.setDocStatus(DocStatus.toCodeOrNull(docStatus));
 
-		final int doctypeID = documentBL.getC_DocType_ID(ctx, docOutboundLog.getAD_Table_ID(), docOutboundLog.getRecord_ID());
-		docOutboundLogLineRecord.setC_DocType_ID(doctypeID);
+		final DocTypeId overrideDocTypeId = archiveBL.getOverrideDocTypeId(ArchiveId.ofRepoId(docOutboundLog.getAD_Archive_ID()));
+		final DocTypeId doctypeID = DocTypeId.ofRepoId(documentBL.getC_DocType_ID(ctx, docOutboundLog.getAD_Table_ID(), docOutboundLog.getRecord_ID()));
+		docOutboundLogLineRecord.setC_DocType_ID(CoalesceUtil.coalesce(overrideDocTypeId, doctypeID).getRepoId());
```

Comment:
use coalesce suppliers or something,
but avoid getting the docTypeID if we already have the overrideDocTypeId provided,
because that's expensive.